integrations/extensions/docs/elasticsearch-install-and-setup/README.md

```diff
@@ -11,19 +11,19 @@ This directory contains documentation for installing and setting up Elasticsearc
 ### Option 1: Set up a custom Elasticsearch extension
 You can follow the guide [here](../../starter-kits/elasticsearch/README.md) to build a custom Elasticsearch extension allowing your assistant to search for information in an Elasticsearch index and show what it finds in the chat.
 ### Option 2: Use the built-in Search integration
-You can set up search over an Elasticsearch index using the built-in search integration. Please follow the instructions [here](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-search-elasticsearch-add) to set up the Search integration for your assistant.  
+You can set up search over Elasticsearch indices using the built-in search integration. Please follow the instructions [here](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-search-elasticsearch-add) to set up the Search integration for your assistant.
 
 By default, keyword search is used for your Search integration, but you can configure the query body in the `Advanced Elasticsearch Settings` to enable advanced search such as semantic search with ELSER, KNN dense vector search, and using nested queries to search over nested documents. Here are some examples:
 
 #### a. Semantic search with ELSER
-For semantic search using ELSER v1, please include the following query body in the `Advanced Elasticsearch Settings`:
+For semantic search using ELSER, please include the following query body in the `Advanced Elasticsearch Settings`:
 
 ```json
 {
   "query":{
     "text_expansion":{
       "ml.tokens":{
-        "model_id":".elser_model_1",
+        "model_id":".elser_model_2",
```

Comment:
Should this be `elser_model_2_linux-x86_64` like the one below?

Reply:
I used different mode IDs and had explanations for them in the notes. But now I think it would be better to just use `elser_model_2_linux-x86_64` as the default for all query bodies, and include in the note that other model IDs can be used if the optimized one is not available.

Reply:
Yes, the list below is good and helpful I think, but I also agree that it's better to have the examples consistent.  Otherwise someone reading casually might get the impression that you need to use the optimized model only with the filter and use the non-optimized model if you don't have a filter, which is not the point that we're trying to make.